package.json

```
@@ -807,6 +908,62 @@
                     "default": "test/runtests.jl",
                     "description": "A workspace relative path to a Julia file that contains the tests that should be run for live testing.",
                     "scope": "window"
+                },
+                "julia.debuggerDefaultCompiled": {
+                    "type": "array",
+                    "default": [
+                        "Base.",
+                        "-Base.!",
+                        "-Base.all",
+                        "-Base.all!",
+                        "-Base.any",
+                        "-Base.any!",
+                        "-Base.atexit",
+                        "-Base.atreplinit",
+                        "-Base.cd",
+                        "-Base.disable_sigint",
+                        "-Base.filter",
+                        "-Base.filter!",
+                        "-Base.findall",
+                        "-Base.findfirst",
+                        "-Base.findlast",
+                        "-Base.findnext",
+                        "-Base.findprev",
+                        "-Base.include_string",
```

Probably we don't want custom iterators to be compiled? or maybe we are ok with that.
```suggestion
                        "-Base.include_string",
                        "-Base.iterate",
```

---

I feel like most people probably don't care about `iterate` (and they can just remove it from the list if they do care), but maybe I'm wrong about that.